src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -41,6 +40,7 @@ const loadSidebarScreen = () => require('../../../pages/home/sidebar/SidebarScre
 const loadValidateLoginPage = () => require('../../../pages/ValidateLoginPage').default;
 const loadLogOutPreviousUserPage = () => require('../../../pages/LogOutPreviousUserPage').default;
 const loadConciergePage = () => require('../../../pages/ConciergePage').default;
+const LoadDesktopSignInRedirectPage = () => require('../../../pages/signin/DesktopSignInRedirectPage').default;
```

Comment:
This also doesn't work

<img width="637" alt="crash2" src="https://github.com/Expensify/App/assets/108292595/b26e6675-f0a9-4c08-9e4c-ea77081d9620">

https://dev.new.expensify.com:8082/desktop-signin-redirect should show below screen but just black page


<img width="1009" alt="desktop redirect" src="https://github.com/Expensify/App/assets/108292595/6f74fcbd-ade4-40ad-881d-dac87732430b">

Reply:
👍🏻